editor/project_manager.cpp

```diff
@@ -2628,11 +2602,22 @@ void ProjectListFilter::_bind_methods() {
 	ADD_SIGNAL(MethodInfo("filter_changed"));
 }
 
+void ProjectListFilter::add_filter_option() {
+	filter_option = memnew(OptionButton);
+	filter_option->set_clip_text(true);
+	filter_option->connect("item_selected", this, "_filter_option_selected");
+	add_child(filter_option);
+
+	has_filter_option = true;
```

Comment:
You added this boolean but you didn't reimplement actual filter options, did you?

Reply:
Fixed, I removed the unused `has_filter_option`.